hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/helper/EntityInterceptorHolder.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.repository.jpa.model.helper;
+
+import java.util.List;
+
+import org.eclipse.hawkbit.repository.model.EntityInterceptor;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * A singleton bean which holds the {@link EntityInterceptor} to have all
+ * interceptors in spring beans.
+ * 
+ */
+public final class EntityInterceptorHolder {
+
+    private static final EntityInterceptorHolder SINGLETON = new EntityInterceptorHolder();
+
+    @Autowired
+    private List<EntityInterceptor> entityInterceptors;
+
+    private EntityInterceptorHolder() {
+
+    }
+
+    /**
+     * @return the entity intreceptor holder singleton instance
+     */
+    public static EntityInterceptorHolder getInstance() {
+        return SINGLETON;
+    }
+
+    public List<EntityInterceptor> getEntityInterceptors() {
```

Comment:
Be careful, you are returning your internal state! Everybody is able to modify it!